Source/WebCore/dom/Document.cpp

```diff
@@ -8780,8 +8780,10 @@ Element* eventTargetElementForDocument(Document* document)
     if (!document)
         return nullptr;
 #if ENABLE(FULLSCREEN_API)
+#if ENABLE(VIDEO)
```

Comment:
Why not use `&&` here instead of nesting `#if`?

Reply:
Updated, thanks for the review!